### PR TITLE
Fixes in labels

### PR DESCRIPTION
**Pull request by evertonfraga**
- [ ] Labels for monorepo packages
- [x] Order of priority labels — use zero-index, because we're programmers :-) (see image 1)
- [x] Unifies label colors (see image 2)

---- 
1) 
image

2) 
image

### CODE REVIEW

File: labels/labels.json [hunk 1]
```diff
@@ -91,10 +95,6 @@
     "name": "prio: P4 nice-to-have",
     "color": "#fde2b4"
   },
-  {
-    "name": "prio: +urgent",
-    "color": "#bf4639"
-  },
```
Comment (holgerd77) [inline labels/labels.json]: This was added intentionally with the purpose of being used as an additional urgency-indicating label (as being marked by the "+" sign). The other labels are capturing the importance of a task. I didn't want to introduce a complete additional category with leveled urgencies, so this was kind of a practical compromise.

Would close this, feel free to re-take on the discussion if something is unclear.

**Comment by holgerd77** [comment]
Ah sorry, re-open, didn't see the other tasks in the task list.

**Comment by holgerd77** [comment]
The differing in the bounty color is also on purpose, since this is logically somewhat a shift from the other bounty labels. The darker colors indicate the process, the one lighter is more somewhat of an additional tagging.

**Comment by holgerd77** [comment]
TBH I also find this zero-index change a bit over the top, this is causing a lot of work to adopt in all the repos, benefits are limited and at the end this is just some question of taste.

**Comment by evertonfraga** [comment]
Right! moving urgent to P0 would be convenient as it would require changes in only one label :-) 

Thanks for the context. So I'll close it for now.

I'll make the third set of changes (monorepo) in a separate PR, so I keep things clean.

**Comment by holgerd77** [comment]
The package labels doesn't really need to be added here, since they are just for the one monorepo, so should just be enough to add them there directly shouldn't it? This can't be reused or is some kind of a standard anyhow.

**Pull request closed without merging**